src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSServiceHelper.cs

```diff
@@ -21,7 +21,7 @@ internal class AWSServiceHelper
     };
 
     internal static string GetAWSServiceName(IRequestContext requestContext)
-        => Utils.RemoveAmazonPrefixFromServiceName(requestContext.Request.ServiceName);
+        => Utils.RemoveAmazonPrefixFromServiceName(requestContext.ServiceMetaData.ServiceId);
```

Comment:
I would suggest giving context for this change in the PR description so people not familiar with AWS SDK details understand this is a safe change.

Reply:
👍 
update PR description above.